README.md

```diff
@@ -78,27 +78,28 @@ with `brew cask install java`
 
 ### Test design
 
-Since we're going to be heavily refactoring and likely removing a lot of code, the focus for now will be on high-level feature tests which will survive that. [cases_test.rb](test/features/cases_test.rb) is an example of the test pattern using Minitest and Capybara which exercises the full stack from a user's point of view.
+Since we're going to be heavily refactoring and likely removing a lot
+of code, the focus for now will be on high-level feature tests which
+will survive that. [cases_test.rb](test/features/cases_test.rb) is an
+example of the test pattern using Minitest and Capybara which
+exercises the full stack from a user's point of view.
 
-### Dependencies
-
-ImageMagick and a global installation of the "Garamond" font are required. On Macs, you can verify the presence of Garamond in Applications > FontBook, and can install ImageMagick via `brew install imagemagick`.
-
-### Javascript
-
-Test scenarios marked with `js: true` will be run in a headless WebKit
+Rails test scenarios marked with `js: true` will be run in a headless WebKit
 environment via Poltergeist. This requires the chromedriver binary to
-be installed, e.g. `brew cask install chromedriver`. Headless tests are significantly slower than static tests, so prefer to avoid writing tests (and features!) that require JS when possible.
+be installed, e.g. `brew cask install chromedriver`. Headless tests
+are significantly slower than static tests, so prefer to avoid writing
+Rails tests (and features!) that require JS when possible.
 
-Guard will not automatically run these tests. This is less than ideal when working on a client-side feature, so you can mark a given test with `focus: true` to force Guard to run it. If no JS tests are enabled, chromedriver will not boot, speeding up the whole test suite considerably.
+### Dependencies
 
-### Guard testing
+ImageMagick and a global installation of the "Garamond" font are required. On Macs, you can verify the presence of Garamond in Applications > FontBook, and can install ImageMagick via `brew install imagemagick`.
```

Comment:
Adding a line like this could be helpful ( it's what I had to do ) 

`or (install it if it's not there)[https://discussions.apple.com/thread/8401159]`